api/src/main/java/io/strimzi/api/kafka/model/CustomResourceConditions.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import io.fabric8.kubernetes.client.CustomResource;
+import io.strimzi.api.kafka.model.status.Status;
+
+import java.util.function.Predicate;
+
+public class CustomResourceConditions {
```

Comment:
We probably want a `private` no-args constructor so no one can instantiate this class.

Reply:
Added private constructor in latest commit